Middleware/SVGAnimations/Opacity.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+
+public class Opacity
+{
+    public Opacity() 
+    {
+        
+    }
+
+    public async Task<string> SVG(string wrapper, IWebHostEnvironment _env, string values, string begin, string duration, string end) 
+    {
+        string svg = await _env.ReadFileFromWebRootAsync("animations/opacity.svg");
+
+        svg = svg.Replace("{{values}}", values);
+        svg = svg.Replace("{{begin}}", begin);
+        svg = svg.Replace("{{duration}}", duration);
+        svg = svg.Replace("{{end}}", end);
```

Comment:
```suggestion
        svg = svg.Replace("{{values}}", values)
            .Replace("{{begin}}", begin);
            .Replace("{{duration}}", duration);
            .Replace("{{end}}", end);
```
This would probably be more idiomatic. Fluency/method chaining is very common in C# (prob due to the prevalance of LINQ which uses that pattern).